operators/src/pro/cache/shared_cache.rs

```diff
@@ -934,27 +936,38 @@ pub trait AsyncCache<C: CacheElement> {
 #[async_trait]
 impl<C> AsyncCache<C> for SharedCache
 where
-    C: CacheElement + ByteSize + Send + 'static,
-    C::Query: Send + Sync,
-    CacheBackend: Cache<C>,
-    CacheQueryEntry<C::Query, C::LandingZoneContainer>: ByteSize,
-    CacheQueryEntry<C::Query, C::CacheContainer>: ByteSize,
-    CacheQueryEntry<C::Query, C::CacheContainer>:
-        From<CacheQueryEntry<C::Query, C::LandingZoneContainer>>,
+    C: CacheElement + Send + Sync + 'static + ByteSize,
```

Comment:
is it possible to simplify this? Mother of all trait bounds.

Reply:
[1b52eeb](https://github.com/geo-engine/geoengine/pull/848/commits/1b52eeb4e5ab835147fac3aff3c3e553d9f53d25)